data/data/nutanix/master/main.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  description = "Created By OpenShift Installer"
+}
+
+provider "nutanix" {
+  wait_timeout = 60
```

Comment:
Is a 60 minute creation timeout reasonable?  I would expect the control plane nodes to be provisioned within a few minutes at most if all is well.  There may be a rationale behind this timeout that I'm not aware of.

Reply:
Good point. We haven't instrumented these stage times and as a result this is just a safe timeout. We can always reduce in future.